fem/fe.hpp

```diff
@@ -504,12 +504,18 @@ class FiniteElement
    /** @brief Given a coefficient and a transformation, compute its projection
        (approximation) in the local finite dimensional space in terms
        of the degrees of freedom. */
+   /** Depending on the derived class that implements this, the degrees of
+       freedom may be computed by interpolation, i.e. evaluation at nodes, which
+       generally differs from L2-projection. */
    virtual void Project(Coefficient &coeff,
```

Comment:
This sounds a bit confusing to me, suggested rewording:

> Depending on the derived class, the approximation can be either local interpolation in the 
degrees of freedom (most common) or local L2 projection (see ExampleClass).

Reply:
Thanks, this is an improvement, but I cannot find any example class that does local L2 projection. After searching fe.cpp, I think every case uses DOF interpolation. There are other `Project` functions that seem to compute an L2 projection matrix, but not this function. I have updated the comments, so please take another look.

Reply:
That sounds good, thanks